model/src/main/java/org/mskcc/cmo/metadb/model/SampleManifestEntity.java

```diff
@@ -155,31 +158,36 @@ public void setUuid(UUID uuid) {
         this.uuid = uuid;
     }
 
-    public List<Sample> getSampleList() {
-        return sampleList;
-    }
-
-    public void setSampleList(List<Sample> sampleList) {
-        this.sampleList = sampleList;
-    }
-
-    /**
-     * Add sample to array.
-     * @param sample
-     */
-    public void addSample(Sample sample) {
-        if (sampleList == null) {
-            sampleList = new ArrayList<>();
-        }
-        sampleList.add(sample);
-    }
-
     public PatientMetadata getPatient() {
         return patient;
     }
 
     public void setPatient(PatientMetadata patient) {
         this.patient = patient;
     }
+    
+    public void setSampleIgoId(Sample s) {
```

Comment:
So the reason why `SampleManifestEntity` was extending `SampleManifest` in the first place was because of our need for these members of `SampleManifest` and to reduce redundancy in the code.